src/components/geometry.js

```diff
@@ -87,12 +87,17 @@ module.exports.Component = registerComponent('geometry', {
     var data = this.data;
     var currentTranslate = previousData.translate || this.schema.translate.default;
     var diff = utils.diff(previousData, data);
-    var geometry = this.el.object3D.geometry;
+    var mesh = this.el.getObject3D('mesh');
```

Comment:
That's an error.  I changed the API but not the placed where it was used. Correcting this now